spk/syncthing-inotify/src/dsm-control.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+# Package
+PACKAGE="syncthing-inotify"
+DNAME="Syncthing-Inotify"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SYNCTHING_DIR="/usr/local/syncthing"
+PATH="${SYNCTHING_DIR}/bin:${PATH}"
+USER="syncthing"
+INOTIFY="${INSTALL_DIR}/bin/syncthing-inotify"
+CONFIG_DIR="${SYNCTHING_DIR}/var"
```

Comment:
Just to check: why use `SYNCTHING_DIR` here, and not a, say, `SYNCTHING_INOTIFY` dir?